packages/dashboard/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="/favicon.ico" />
+    <link rel="icon" href="favicon.ico" />
```

Comment:
```suggestion
    <link rel="icon" href="%BASE_URL%favicon.ico" />
```

As shown in vite docs, seems that it expects base url to always end with a trailing slash https://vitejs.dev/config/shared-options#base. So we may need to update the app-config, we can also probably simplify `url.ts` with this assumption.

Reply:
thanks for the reference, changed in https://github.com/open-rmf/rmf-web/pull/989/commits/8e6d672b0ea95b93849d97a85723d0653f5505e0